pkg/render/compliance.go

```diff
@@ -156,6 +159,9 @@ func (c *complianceComponent) Objects() ([]runtime.Object, []runtime.Object) {
 	}
 
 	complianceObjs = append(complianceObjs, secretsToRuntimeObjects(CopySecrets(ComplianceNamespace, c.esSecrets...)...)...)
+	if c.managerSecret != nil {
```

Comment:
See Line 545 (AnnotationHash) for the annotation that we need to add for this Secret.